lib/yaml_db/serialization_helper.rb

```diff
@@ -57,16 +57,21 @@ def reenable_logger
 
     class Load
       def self.load(io, truncate = true)
-        ActiveRecord::Base.connection.transaction do
+        connection.transaction do
           load_documents(io, truncate)
         end
       end
 
       def self.truncate_table(table)
         begin
-          ActiveRecord::Base.connection.execute("TRUNCATE #{Utils.quote_table(table)}")
+          if connection.adapter_name == "SQLite"
+            connection.execute("DELETE FROM #{Utils.quote_table(table)}")
+            connection.execute("VACUUM")
```

Comment:
Given the performance penalty, is `VACUUM` appropriate to do here? Aren't we always refilling the database (not necessarily with as much data, I'll admit)?

Plus, this kind of optimization generally seems a bit out of scope for what yaml_db is doing.